docs/source/tools/trigger-service/index.rst

```diff
@@ -352,3 +352,26 @@ HTTP Response
 -------------
 
 A status code of ``200`` indicates a successful readiness check.
+
+
+Metrics
+*******
+
+Enable and Configure Reporting
+==============================
+
+To enable metrics and configure reporting, you can use the below config block in application config:
+
+.. code-block:: none
```

Comment:
Can we change this example to be 'Prometheus' so the user has an example for the format that we want them to use?

Reply:
Good point — changed.